scripts/ovale_monk_spells.lua

```diff
@@ -294,7 +300,7 @@ Define(tigereye_brew_use_buff 116740)
 	SpellInfo(tigereye_brew_use_buff duration=15)
 Define(touch_of_death 115080)
 	SpellInfo(touch_of_death cd=120 tag=main)
-	SpellAddDebuff(touch_of_death touch_of_death_debuff=1)
+	SpellAddTargetBuff(touch_of_death touch_of_death_debuff=1)

```

Comment:
You are totally right. I changed the relevant spell info to SpellAddTargetDebuff (see https://github.com/Sidoine/Ovale/pull/343/commits/2a6a802b74023e970393b861fef540bb367fe910)